### PR TITLE
allow easier selection of os-specific sections

### PR DESCRIPTION
**Pull request by evgeni**
* use css classes for selection, allowing multiple sections to be marked
  as "to show" or "to hide"
* the above requires the "show/hide" loop to be split in two:
  first hide everything, then show what the author selected
* use this feature to hide tracer on EL5/6 and yum instructions on SUSE

### CODE REVIEW

**Comment by evgeni** [comment]
review note: before merging, I'd like to apply the same change to katello 3.8/3.9/3.10 docs, not only nightly, but was too lazy before the review :)

File: plugins/katello/nightly/installation/clients.md [hunk 1]
```diff
@@ -87,17 +88,22 @@ We generally recommend using Foreman Remote Execution or Ansible for remote acti
...
 Optionally you can also install `katello-host-tools-tracer` and the client will report processes that need restarting after an update back to the Katello server.
 
 {% highlight bash %}
 yum install katello-host-tools-tracer
 {% endhighlight %}
+</div>
+</div>
 
-For Suse Clients, only katello-host-tools is supported:
+<div class="sles11 sles12" style="display:none;" markdown="1">
```
Comment (ekohl) [inline plugins/katello/nightly/installation/clients.md:100]: What's the `markdown=1` part?

Comment (evgeni) [reply plugins/katello/nightly/installation/clients.md:100]: I don't know, I copied it over from the other sections.

I *think* it indicates that the content is markdown and not plain HTML

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
Of course I read your note the first time I reviewed but not the second time ...

**Pull request merged**